### PR TITLE
Pulling changes from xhyve

### PR DESCRIPTION
**Pull request by frezbo**
Signed-off-by: Noel Georgi <git@frezbo.com>

Fixes: #236 @ijc

### CODE REVIEW

**Comment by ijc** [comment]
Thanks. I expect CI will fail due to the issue being address in #235 and this will need a rebase once that is fixed.

Do I understand correctly that the MAC is derived from the UUID and so if you keep the UUID constant you can run again without `-M` to get an actual VM?

By any chance is there an `xhyve` commit which updates the `xhyve.1` manpage (from which `hyperkit.1` is derived) with this option?

**Comment by frezbo** [comment]
@ijc I'm not sure about your question, I just started looking at the `xhyve` and `hyperkit` code. I found this issue when using the `vagrant-xhyve` (https://github.com/oldpatricka/vagrant-xhyve) plugin with hyperkit. I assume the man page is there in `xhyve` If so I will update.

**Comment by frezbo** [comment]
@ijc Seems the man page was never updated: https://github.com/machyve/xhyve/commit/c6bf5fcad6bbb2bb9023180199a771752bde64ff#diff-895d2395602688834afea736f799fd48. I will update the missing code as per this commit

File: hyperkit.1 [hunk 1]
```diff
@@ -100,6 +100,10 @@ The size argument may be suffixed with one of K, M, G or T (either upper
 or lower case) to indicate a multiple of kilobytes, megabytes, gigabytes,
 or terabytes.
 If no suffix is given, the value is assumed to be in megabytes.
+and
+.Ar com3.
```
[the remark this answers is not in the document]
Comment (frezbo) [reply hyperkit.1]: yeh, I'm still new to `man` syntax and the macros 😂

[the remark this answers is not in the document]
Comment (frezbo) [reply hyperkit.1]: fixed

**Comment by ijc** [comment]
@frezbo please could you rebase to master to pickup the CI fix from#240, thanks and sorry for the long delay.

**Comment by frezbo** [comment]
@ijc Already rebased when the fix was merged to master

**Comment by ijc** [comment]
> @ijc Already rebased when the fix was merged to master

Oops, so you did, for some reason my favicon for this tab was showing the red cross for CI failure not the green tick for pass (sometimes it seems to get stuck and not pickup updates).

I'll merge, thanks for your contribution!

**Pull request merged**